scipy/spatial/distance.py

```diff
@@ -1084,7 +1084,7 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
 
        .. math::
 
-            d(u,v) = \\frac{\\sum_i {u_i-v_i}}
+            d(u,v) = \\frac{|\\sum_i {u_i-v_i}|}
```

Comment:
the absolute value symbols should be directly around ``u_i-v_i``, not outside ``\sum``.